sdk/core/azure-core-http-netty/src/main/java/com/azure/core/http/netty/implementation/NettyVersion.java

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.core.http.netty.implementation;
+
+import com.azure.core.util.CoreUtils;
+import com.azure.core.util.logging.ClientLogger;
+import io.netty.util.Version;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * Information about the versions of Netty libraries used at runtime.
+ */
+public final class NettyVersion {
```

Comment:
Could this be a package-private class in com.azure.core.http.netty rather than a public one in the implementation package? I don't see this class used elsewhere other than this builder class.  

I am generally hesitant to make something public even if it is in the 'implementation' package as it doesn't stop people from referencing it (in older jdks).

Reply:
I can remove the class and move this function to `Utility` in implementation as that already exists.